.github/workflows/cron.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Trivy Java DB
+on:
+  schedule:
+    - cron: "0 0 * * 3" # update indexes every Thursday in 00:00
+  workflow_dispatch:
+env:
+  GH_USER: aqua-bot
+  TRIVY_JAVA_DB_CACHE: cache
+jobs:
+  build:
+    name: Build DB
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Build the binary
+        run: make build
+
+      - name: Build database
+        run: make db-build
+
+      - name: Compress database
+        run: make db-compress
+
+      - name: Move DB
+        run: mv cache/trivy-java-db/java-db/db.tar.gz .
+
+      - name: Login to GitHub Packages Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ env.GH_USER }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install oras
+        run: |
+          # oras was rollbacked to v0.12.0, because now v0.13.0 (the latest version) contains bugs: https://github.com/oras-project/oras/issues/447
```

Comment:
Seems like this issue is closed. Can you try the newer version?

Reply:
created #4 for updating oras